src/sync/outgoingSyncUtils.js

```diff
@@ -417,6 +417,18 @@ const generateSyncData = (settings, recordType, record) => {
         data: record.data,
       };
     }
+    case 'AdverseDrugReaction': {
+      return {
+        ID: record.id,
+        form_schema_ID: record.formSchema?.id ?? '',
+        name_ID: record.name?.id ?? '',
+        user_ID: record.user?.id ?? '',
+        store_ID: settings.get(THIS_STORE_ID),
+        entry_date: getDateString(record.entryDate),
+        entry_time: getTimeString(record.entryDate),
+        data: record._data,
```

Comment:
shouldn't this be `data`?

Reply:
🤔  `_data` is the string so I think we send it like this to be parsed on the other side

Reply:
Should we change `NameNote` then? that one is sending the object

Reply:
ah.. hmm.. I do remember testing this against the msupply branch and receiving the data, but I will make sure to give this stringifying a proper test again